cmd/captain/quarantine.go

```diff
@@ -44,6 +50,10 @@ func AddQuarantineFlags(rootCmd *cobra.Command, cliArgs *CliArgs) {
 					case "github-step-summary":
 						stepSummaryPath := os.Getenv("GITHUB_STEP_SUMMARY")
 						if stepSummaryPath == "" {
+							captain.Log.Debug(
+								"Skipping configuration of the 'github-step-summary' reporter " +
```

Comment:
not what I had in mind but also fine.

Reply:
Cool- if it's not a huge deal for now, I'll probably leave it like this and we can circle back with a larger refactor later

Reply:
"not a huge deal" mostly meaning "this won't break anything" 😅